### PR TITLE
Bank 2 dump

### PR DESCRIPTION
**Pull request by Pokeglitch**
Dumped all of bank 2, and I think I converted all of the address to labels

The binaries seem to be corrupted data. some graphics are also corrupted.

### CODE REVIEW

File: engine/dumps/bank02.asm [hunk 1]
```diff
@@ -0,0 +1,3164 @@
...
+Function8438: ; 02:4438
+	ld hl, QuestionEmoteGFX
+	jr Function8440
+
+Function843d: ; 02:443d
+	ld hl, HappyEmoteGFX
+
+Function8440: ; 02:4440
+	ld de, vChars1 + $780
+	ld b, $04
```
Comment (luckytyphlosion) [inline engine/dumps/bank02.asm]: Many missed references to hardcoded banks. (`ld b, $nn` for video copy functions)

File: engine/dumps/bank02.asm [hunk 2]
```diff
@@ -0,0 +1,3164 @@
...
+	
+Function8786: ; 02:4786
+	ld a, [wFlyDestination]
+	push af
+	xor a
+	ld [wFlyDestination], a
+	call Function881e
+	ld de, PokedexNestIconGFX
+	ld hl, vChars0 + $7f0
+	ld bc, $0201
```
Comment (luckytyphlosion) [inline engine/dumps/bank02.asm]: Hardcoded bank.

File: engine/dumps/bank02.asm (hunk 1 above)
Comment (Pokeglitch) [reply engine/dumps/bank02.asm]: b is the size of the gfx, c is the bank

File: engine/dumps/bank02.asm (hunk 2 above)
Comment (Pokeglitch) [reply engine/dumps/bank02.asm]: Fixed

**Pull request merged**